tests/monitoring/monitoring.go

```diff
@@ -124,33 +126,25 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			originalKv := util.GetCurrentKv(virtClient)
 			kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
-			kv.Spec.CustomizeComponents = v1.CustomizeComponents{
-				Patches: []v1.CustomizeComponentsPatch{
-					{
-						ResourceName: virtHandler.deploymentName,
-						ResourceType: "DaemonSet",
-						Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
-						Type:         v1.StrategicMergePatchType,
-					},
+
+			patchData := []map[string]interface{}{
```

Comment:
Also here, I'd suggest to use functions from `patch.go`:
```
patchData, err := patch.New(patch.WithAdd("/spec/template/spec/nodeSelector/kubernetes.io~1hostname", "does-not-exist")).GeneratePayload()
Expect(err).ToNot(HaveOccurred())
```

Reply:
Done